cmd/fyne/internal/util/file.go

```diff
@@ -27,6 +27,22 @@ func CopyExeFile(src, tgt string) error {
 	return copyFileMode(src, tgt, 0755)
 }
 
+// WriteFile write the content of data into a regular file, target.
+func WriteFile(target string, data []byte) error {
```

Comment:
I would recommend just using `ioutil.WriteFile` instead of adding a new helper

Reply:
Oh, I didn't found that one. I only saw io.WriteFile which wasn't available in 1.14. Will switch to that.

Reply:
Yeah, the older `ioutil` package is being replaced by new `io` features and it will go away in the future - but not before Go 2.0 :)